city_scrapers/spiders/chi_ssa_52.py

```diff
@@ -0,0 +1,168 @@
+        needs.
+        """
+        items = response.css("div.paragraph")[3:4]
+        titles = items.css("strong::text").getall()
+        meetings = items.css("ul")
+        items = list(zip(titles, meetings))
+
+        for i, item in enumerate(items):
+            for meet in item[1].css("li"):
+                meet = self._clean_meet(meet)
+
+                meeting = Meeting(
+                    title=self._parse_title(item[0]),
+                    description=self._parse_description(item),
+                    classification=self._parse_classification(item),
+                    start=self._parse_start(meet),
+                    end=self._parse_end(item),
+                    all_day=self._parse_all_day(item),
+                    time_notes=self._parse_time_notes(item),
+                    location=self._parse_location(item),
+                    links=self._parse_links(item),
+                    source=self._parse_source(response),
+                )
+
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+
+                yield meeting
+
+    def _replace_meet_chars(self, meet):
+        meet = meet.css("::text").get()
+        meet = meet.replace(".", "")
+        meet = meet.replace("\xa0", "")
+        meet = meet.replace("…", "")
+        meet = meet.replace(",", "")
+        meet = meet.strip()
+
+        return meet
+
+    def _clean_meet(self, meet):
+        """Clean and parse a possible meet <li> item"""
+        payload = {}
+        meet = self._replace_meet_chars(meet)
+        meet = meet.split()
+
+        if len(meet) == 4:
+            day_week = meet[0]
+            month = meet[1]
+            month_day = meet[2]
+            year = meet[3][:4]
+            time = meet[3][4:]
+            payload = {
+                "day_week": day_week,
+                "month": month,
+                "month_day": month_day,
+                "year": year,
+                "time": time,
+            }
+
+        if len(meet) == 6:
+            day_week = meet[0]
+            month = meet[1]
+            month_day = meet[2]
+            year = meet[3][:4]
+            time = meet[3][4:]
+            info = " ".join(meet[4:])
+            payload = {
+                "day_week": day_week,
+                "month": month,
+                "month_day": month_day,
+                "year": year,
+                "time": time,
+                "info": info,
+            }
+
+        return payload
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        item = item.replace("\xa0", "").strip()
+        return item
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        return ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return NOT_CLASSIFIED
```

Comment:
We can use `COMMISSION` for all of these meeting since it seems like there aren't committees

Reply:
Thanks, i'll change it.